doc/classes/@GDScript.xml

```diff
@@ -100,6 +111,11 @@
 			<argument index="0" name="s" type="float">
 			</argument>
 			<description>
+				Returns the arc tangent of 's' in radians. Use to get the angle of tangent 's'. Notice that because of the sign ambiguity, the function cannot determine with certainty in which quadrant the angle falls only by its tangent value. See [method atan2] for an alternative that takes a fractional argument instead.
```

Comment:
Use method instead of function. Suggestion:
`Returns the arc tangent of 's' in radians. Use it to get the angle from an angle's tangent in trigonometry: [code]atan(tan(angle)) == angle[/code]. The method cannot know in which quadrant the angle should fall. See [method atan2] if you always want an exact angle.`